receiver/otlpreceiver/otlp_test.go

```diff
@@ -178,10 +178,11 @@ func TestJsonHttp(t *testing.T) {
 		},
 	}
 	addr := testutil.GetAvailableLocalAddress(t)
+	tracesURLPath := "/v1/traceingest"
 
 	// Set the buffer count to 1 to make it flush the test span immediately.
 	sink := &errOrSinkConsumer{TracesSink: new(consumertest.TracesSink)}
-	ocr := newHTTPReceiver(t, addr, sink, nil)
+	ocr := newHTTPReceiver(t, addr, tracesURLPath, defaultMetricsURLPath, sink, nil)
```

Comment:
I might find the answer later in this PR, but why is it missing the logs URL path?

Reply:
Don't think logs were ever really leverage in this test file.  Of course, them being present in the config below proves there is some need, so I'll just go ahead and add this additional parameter to avoid confusion.